<launch>
  <!--Include description and control launch files-->
  <include file="$(find training_data_generator)/launch/robot_description.launch"/>
  <include file="$(find training_data_generator)/launch/robot_control.launch"/>

  <!--Launch a gazebo world-->
  <include file="$(find gazebo_ros)/launch/empty_world.launch"/>

  <!--spawn a robot in gazebo world-->
  <node name="urdf_spawner" pkg="gazebo_ros" type="spawn_model" respawn="false" output="screen"
  args="-urdf -param robot_description -x 0 -y 1.8 -z 0 -R 0 -P 0 -Y 0 -model training_data_generator"/>

  <!-- cloud transformer-->
  <!--<node name="cloud_transformer" pkg="training_data_generator" type="cloud_transformer" respawn="false"/>-->

  <!-- The feature extractor node -->
  <node name="feature_extractor" pkg="training_data_generator" type="feature_extractor" respawn="false"/>

  <!--<node name="spawn_beer" pkg="gazebo_ros" type="spawn_model" respawn="false" output="screen"
  args="-x 0.0 -y 1.0 -z 1.0 -file $(find training_data_generator)/models/beer/model.sdf -sdf -model beer"/>-->

</launch>
